tests/flags/flags.go

```diff
@@ -99,6 +100,7 @@ func init() {
 	flag.StringVar(&DNSServiceName, "dns-service-name", "kube-dns", "cluster DNS service name")
 	flag.StringVar(&DNSServiceNamespace, "dns-service-namespace", "kube-system", "cluster DNS service namespace")
 	flag.StringVar(&MigrationNetworkNIC, "migration-network-nic", "eth1", "NIC to use on cluster nodes to access the dedicated migration network")
+	flag.StringVar(&MigrationNetworkName, "migration-network-name", "", "name of a network attachment definition to be used on a dedicated migration network tests")
```

Comment:
nit: The NAD is a CRD name, I think text should imply that by saying `network-attachment-definition CR name` or `NetworkAttachmentDefintion CR name` 
```suggestion
	flag.StringVar(&MigrationNetworkName, "migration-network-name", "", "name of the NetworkAttachmetDefintion CR to be used for dedicated migration network tests")
```

Reply:
fixed